models/bamf_nnunet_mr_liver/meta.json

```diff
@@ -0,0 +1,135 @@
+{
+    "id": "",
+    "name": "bamf_nnunet_mr_liver",
+    "title": "AIMI MRI Liver",
+    "summary": {
+        "description": "An nnU-Net based model to segment liver from T1 weighted MRI scans",
+        "inputs": [
+            {
+                "label": "Input Image",
+                "description": "The abdominal T1 weighted  MRI scan of a patient.",
+                "format": "DICOM",
+                "modality": "MR",
+                "bodypartexamined": "LIVER",
+                "non-contrast": true,
+                "contrast": false
+            }
+        ],
+        "outputs": [
+            {
+                "label": "Segmentation",
+                "type": "Segmentation",
+                "description": "Segmentation liver",
+                "classes": [
+                    "LIVER"
+                ]
+            }
+        ],
+        "model": {
+            "architecture": "U-net",
+            "training": "Supervised",
+            "cmpapproach": "3D, ensemble"
+        },
+        "data": {
+            "training": {
+                "vol_samples": 350
+            },
+            "evaluation": {
+                "vol_samples": 7
+            },
+            "public": true,
+            "external": true
+        }
+    },
+    "details": {
```

Comment:
`cite` is missing.

Reply:
Apparently there are different rules depending on the country, here we use license as the verb and licence as the noun ^^

Knowing this, using license as keyword likely would've been the easier choice, but for now could you just change `license` to `licence` so the checks pass :)?